### PR TITLE
RHDEVDOCS-4479: update install guides for odo 30

### PR DESCRIPTION
**Pull request by gabriel-rh**
Version(s):
4.8+

Issue:
https://issues.redhat.com/browse/RHDEVDOCS-4479

Link to docs preview:
https://51827--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/developer_cli_odo/installing-odo.html

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: cli_reference/developer_cli_odo/installing-odo.adoc [hunk 1]
```diff
@@ -8,7 +8,9 @@ toc::[]
 
 // The following section describes how to install `{odo-title}` on different platforms using the CLI or the Visual Studio Code (VS Code) IDE.
 
-You can install the `{odo-title}` CLI on Linux, Windows, or macOS by downloading a binary. You can also install the OpenShift VS Code extension, which uses both the `{odo-title}` and the `oc` binaries to interact with your OpenShift Container Platform cluster. For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
+You can install the `{odo-title}` CLI on Linux, Windows, or macOS by downloading a binary. 
+
+//You can also install the OpenShift VS Code extension, which uses both the `{odo-title}` and the `oc` binaries to interact with your OpenShift Container Platform cluster. For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
```
Comment (rolfedh) [inline cli_reference/developer_cli_odo/installing-odo.adoc:13]: I'm guessing this text is describing Red Hat's [OpenShift Connector](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector). If so, what do you think about: 
```suggestion
You can also install the OpenShift Connector extension for Microsoft Visual Studio Code, which uses both the `{odo-title}` and the `oc` binaries to interact with your {product-title} cluster. For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
```

File: cli_reference/developer_cli_odo/installing-odo.adoc [hunk 2]
```diff
@@ -8,7 +8,9 @@ toc::[]
 
 // The following section describes how to install `{odo-title}` on different platforms using the CLI or the Visual Studio Code (VS Code) IDE.
 
-You can install the `{odo-title}` CLI on Linux, Windows, or macOS by downloading a binary. You can also install the OpenShift VS Code extension, which uses both the `{odo-title}` and the `oc` binaries to interact with your OpenShift Container Platform cluster. For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
+You can install the `{odo-title}` CLI on Linux, Windows, or macOS by downloading a binary.
```
Comment (rolfedh) [inline cli_reference/developer_cli_odo/installing-odo.adoc:11]: This is an out-of-scope topic, but you might want to consider having separate attributes for the product title and the command. 
For example, https://docs.openshift.com/container-platform/4.11/cli_reference/developer_cli_odo/understanding-odo.html refers to the product as 
```
Red Hat OpenShift Developer CLI (`odo`)
``` 
I also see many instances of the literal command like this throughout the docs. For example:
```
Required for `odo` CLI.
```
Whatever you choose, you might want to make it consistent across the docs. Without that consistency, the benefit of having a command attribute is limited.

File: cli_reference/developer_cli_odo/installing-odo.adoc (hunk 1 above)
Comment (rolfedh) [reply cli_reference/developer_cli_odo/installing-odo.adoc:13]: Use {product-title} to refer to OCP.

File: modules/developer-cli-odo-installing-odo-on-linux.adoc [hunk 3]
```diff
@@ -7,27 +7,28 @@
...
+|Linux on IBM Power|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le[`odo-linux-ppc64le`] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le.tar.gz[`odo-linux-ppc64le.tar.gz`]
+|Linux on IBM Z and LinuxONE|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x[`odo-linux-s390x`] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x.tar.gz[`odo-linux-s390x.tar.gz`]
 |===
 
 
 .Procedure
 
-. Navigate to the link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[content gateway] and download the appropriate file for your operating system and architecture.
+. Browse to the link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[content gateway] and download the appropriate file for your operating system and architecture.
```
Comment (rolfedh) [inline modules/developer-cli-odo-installing-odo-on-linux.adoc:24]: See the IBM SG "Referring to web addresses" section:

Use the following verbs and prepositions with web addresses:
To direct readers to a website or web page, use see or go to.
For more information about the product, see CICS Explorer
.
To obtain videos for software support, go to the IBM Software Subscription and Support
website.
To describe where information is located, use on.
See the latest product information on the CICS Explorer
website.
Find information about database management systems on the IBM Db2 Database
page.
To describe where to download a file, use from.
Download the video from the IBM Software Subscription and Support
website.

File: modules/developer-cli-odo-installing-odo-on-macos.adoc [hunk 4]
```diff
@@ -7,24 +7,25 @@
...
+|Operating system|Binary|Tar file
+|macOS on AMD|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-darwin-amd64[odo-darwin-amd64]|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-darwin-amd64.tar.gz[odo-darwin-amd64.tar.gz]
+|macOS on ARM|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-darwin-arm64[odo-darwin-arm64]|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-darwin-arm64.tar.gz[odo-darwin-arm64.tar.gz]
 |===
 
 .Procedure
 
-. Navigate to the link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[content gateway] and download the appropriate file:
+. Browse to the link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[content gateway] and download the appropriate file:
```
Comment (rolfedh) [inline modules/developer-cli-odo-installing-odo-on-macos.adoc:21]: See IBM SG "Referring to web addresses" information.

**Review by rolfedh — APPROVED** [review]
Hi Gabe, I left some discussion questions and style nits. Great work!

File: cli_reference/developer_cli_odo/installing-odo.adoc (hunk 1 above)
Comment (rolfedh) [reply cli_reference/developer_cli_odo/installing-odo.adoc:13]: ^^ This is the only non-nit.

Comment (gabriel-rh) [reply cli_reference/developer_cli_odo/installing-odo.adoc:13]: That paragraph is commented out entirely, as the functionality has been removed upstream docs. I will delete the content once an SME confirms that it is ok to remove downstream.

If it comes back in, I will make the changes indicated.

File: cli_reference/developer_cli_odo/installing-odo.adoc (hunk 2 above)
Comment (gabriel-rh) [reply cli_reference/developer_cli_odo/installing-odo.adoc:11]: The OpenShift CLI is never referred to with the Red Hat prefix, so to be consistent I guess I should remove that one instance of Red Hat OpenShift Developer CLI and just call it "OpenShift Developer CLI" there.

So the question then is, do I use  "OpenShift Developer CLI (`odo`)" everywhere? I'm not sure, as `odo` is not an acromyn, so not as clear cut as oc - 

Also odo is an upstream project where it is also a Kubernetes developer CLI as well as OpenShift (whereas oc is OpenShift-only) and the acronym hasn't any meaning whatsoever...

So I am tempted to use plain `odo` everywhere, and reword the understanding-odo.html to say:

> `odo`, the developer CLI for OpenShift and Kubernetes....

Thoughts?

Other examples:
Installing {pipelines-title} CLI (tkn) on Linux using an RPM...
The `opm` CLI tool is provided by the Operator Framework for use with t...
You can customize your Knative (`kn`) CLI setup by creat...

File: modules/developer-cli-odo-installing-odo-on-linux.adoc (hunk 3 above)
Comment (gabriel-rh) [reply modules/developer-cli-odo-installing-odo-on-linux.adoc:24]: Ok, I will change to "Go to the content gateway and download...." seeing as that is what the IBM SG says

File: modules/developer-cli-odo-installing-odo-on-macos.adoc (hunk 4 above)
Comment (gabriel-rh) [reply modules/developer-cli-odo-installing-odo-on-macos.adoc:21]: Ok, I will change to "Go to the content gateway and download...." seeing as that is what the IBM SG says

File: cli_reference/developer_cli_odo/installing-odo.adoc (hunk 1 above)
Comment (rolfedh) [reply cli_reference/developer_cli_odo/installing-odo.adoc:13]: Better to complete the SME review task before the peer review task.

Comment (gabriel-rh) [reply cli_reference/developer_cli_odo/installing-odo.adoc:13]: In a perfect world, yes. I am waiting on SME reviews for a lot of content and had hoped that putting the cart before the horse and asking for peer review in the mean-time might speed things up...

File: cli_reference/developer_cli_odo/installing-odo.adoc (hunk 2 above)
Comment (gabriel-rh) [reply cli_reference/developer_cli_odo/installing-odo.adoc:11]: I will change the title in the updating-odo file as outlined above, but that is in a separate PR.

I will create a new PR to handle the inconsistencies between using `{odo-title}` variable and `odo` across the docs.

File: modules/developer-cli-odo-installing-odo-on-macos.adoc (hunk 4 above)
Comment (gabriel-rh) [reply modules/developer-cli-odo-installing-odo-on-macos.adoc:21]: Done

File: modules/developer-cli-odo-installing-odo-on-linux.adoc (hunk 3 above)
Comment (gabriel-rh) [reply modules/developer-cli-odo-installing-odo-on-linux.adoc:24]: Done

**Pull request closed without merging**